### ISSUE TITLE
Lesson 5 part 8: Name That Element! typo

### ISSUE THREAD

**Issue by pekac45** [udacity/ud891#66]
First excercise says: 

`<button aria-lavbel="Gumnut">
  Eucalyptus
</button>`

but there is a typo in aria-lavbel which should be aria-label
